.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -31,8 +34,38 @@ steps:
     key: "agentbeat-mandatory-tests"
 
     steps:
+
+      - label: ":windows: x-pack/agentbeat: Win 2022 System Tests"
+        key: "mandatory-win-2022-system-tests"
+        command: |
+          # .buildkite/scripts/gcp_auth.ps1
```

Comment:
@michalpristas is this commented out on purpose?
I see the following failure which might be related: https://buildkite.com/elastic/beats-xpack-agentbeat/builds/7836#0195137e-6ead-45f3-8641-1919a716408b/122-130

Reply:
yes this file is not here, i need to do some other backports and uncomment this

Reply:
backports done, uncommented